docs/guides/gunw_product_guide.md

```diff
@@ -22,28 +26,24 @@ You can download ARIA-S1-GUNW products from the Alaska Satellite Facility’s (A
 
 You can also use the Vertex SBAS tool to download networks of interferograms for a specific location. See [this guide](https://docs.asf.alaska.edu/vertex/sbas/){target=_blank} for more information.
 
-**NOTE: ARIA S1 GUNW products are not produced globally! If you cannot find ARIA S1 GUNW for your area of interest, see the On-Demand section below.**
-
-### Creating ARIA S1 GUNWs using HyP3
-If ARIA S1 GUNWs are not available for your area or reference/secondary date combination of interest, you can create new ARIA S1 GUNWs using the HyP3 on-demand system.
+{% endblock %}
 
-**Note: The lists of reference and secondary granules and the ARIA frame ID must be carefully selected to result in a valid ARIA product. See the [framing](#aria-frame-ids "Jump to framing section of this document") section for more details.**
+{% block novertex %}
```

Comment:
Is there any way we can trick the template into not including the Vertex section? I could also remove the screenshot from the Accessing Existing GUNW Products, but the two screenshots by each other are a bit confusing. 
![Screenshot 2025-02-27 at 3 44 22 PM](https://github.com/user-attachments/assets/941d9da8-6d99-4cd0-ae82-c9fcebe5f1f7)